src/components/header.tsx

```diff
@@ -1,18 +1,51 @@
 import type { NextPage } from 'next';
+import Link from 'next/link'
+
 
 const Header: NextPage = () => {
-    const style = {
-        backgroundColor: 'blue',
-        color: 'white',
-        width: '100%',
-        height: '50px'
-    }
     return (
-        <div className='header' style={style}>
-            <div style={{ margin: '10px' }}>
-                CS: 498 - SAP
+        <nav className="sticky p-6 mx-auto bg-white w-full">
```

Comment:
In one of our meetings, I remembered someone saying that the navigation bar will only be in the landing page. If so, should this component strictly be stored in `pages/index.tsx`? Might need more clarification from @aasthabhatt00. `header.tsx` should be used as a template header for all the headers of our pages, similar to `footer.tsx`.

Reply:
Where and how can I add a description for the changes I made in this PR? Is there a section in the GitHub PR where I can specifically write a summary?

Reply:
Your top most comment is your PR description, you can edit that by clicking on the three buttons located at the top right of the comment.

Reply:
I was able to figure out how to add the description to the PR, thanks!